cmd/kubeapps-apis/cmd/root.go

```diff
@@ -65,10 +66,14 @@ func Execute() {
 }
 
 func init() {
+	log.InitFlags(nil)
 	cobra.OnInitialize(initConfig)
 	rootCmd = newRootCmd()
 	rootCmd.SetVersionTemplate(version)
 	setFlags(rootCmd)
+        //set initial value of verbosity
+	goflag.Set("v", "3")
```

Comment:
Out of curiosity, how did you come up with this default number? In other projects I work on we use verbosity level default to 2 but to be fair I've never got a definitive answer on why.

Thanks!